src/options/getRepoOptions.ts

```diff
@@ -0,0 +1,40 @@
+import { cosmiconfigSync } from 'cosmiconfig';
+import { getDefaultRemoteBranch } from 'workspace-tools';
+import { RepoOptions, CliOptions } from '../types/BeachballOptions';
+
+export function getRepoOptions(cliOptions: CliOptions): RepoOptions {
+  let repoOptions: RepoOptions | null;
+  if (cliOptions.configPath) {
+    repoOptions = tryLoadConfig(cliOptions.configPath);
+    if (!repoOptions) {
+      console.error(`Config file "${cliOptions.configPath}" could not be loaded`);
+      process.exit(1);
+    }
+  } else {
+    repoOptions = trySearchConfig();
+  }
+  repoOptions = repoOptions || ({} as RepoOptions);
+
+  if (!cliOptions.branch) {
+    // Add or fix up the branch in repoOptions if it's not specified in cliOptions (which takes precedence)
+    if (repoOptions.branch && !repoOptions.branch.includes('/')) {
+      repoOptions.branch = getDefaultRemoteBranch(repoOptions.branch, cliOptions.path);
+    } else if (!repoOptions.branch) {
+      repoOptions.branch = getDefaultRemoteBranch('master', cliOptions.path);
+    }
+  }
```

Comment:
This is the bit that was added compared with the old implementation in `getRootOptions.ts`, am I correct? Could you explain why we check for existence of '/' in the branch name?